nautobot/docs/installation/install_system.md

```diff
@@ -0,0 +1,475 @@
+# Virtual Machine / Host Install
+
+The documentation assumes that you are running one of the following:
+
+- Ubuntu 20.04+
+- Debian 11+
+- RHEL/CentOS 8.2+
+    - Also includes other derivatives of RHEL such as RockyLinux or AlmaLinux
+
+## Install System Packages
+
+Install the prerequisite system libraries and utilities.
+
+This will install:
+
+- Git
+- Python 3
+- Pip
+- Redis server and client
+
+=== "Ubuntu/Debian"
+
+    ```no-highlight
+    sudo apt update -y
+    sudo apt install -y git python3 python3-pip python3-venv python3-dev redis-server
+    ```
+
+=== "RHEL8 + Derivatives"
+
+    ```no-highlight
+    sudo dnf check-update
+    sudo dnf install -y git python38 python38-devel python38-pip redis
+    ```
+
+## Database Setup
+
+In this step you'll set up your database server, create a database and database user for use by Nautobot, and verify your connection to the database.
+
+You must select either MySQL or PostgreSQL. PostgreSQL is used by default with Nautobot, so if you just want to get started or don't have a preference, please stick with PostgreSQL.
```

Comment:
Not blocking, but might be good to add a note indicating that MariaDB is *not* supported as a MySQL-like database in Nautobot 1.x. (We need to retest for 2.x now that we've dropped `django-mptt` which was the main incompatibility with MariaDB).